autocertkit/network_tests.py

```diff
@@ -152,6 +152,7 @@ def configure_routes(self):
     def run(self):
         """This classes run test function"""
         self.deploy_iperf()
+        time.sleep(30)
```

Comment:
This is probably okay. But it little bit scares me that we just put some sleep without knowing exact reason.
If you know exact reason, then can we put some sort of check routine? i.e. wait for 30 secs until iperf deployed properly.